pallets/nfts/src/lib.rs

```diff
@@ -19,6 +19,13 @@ use sp_runtime::{DispatchResult, RuntimeDebug};
 use sp_std::result;
 use ternoa_common::traits::{LockableNFTs, NFTs};
 
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Default, RuntimeDebug)]
```

Comment:
I'd recommend deriving `Default` too if you can, this will be useful in benchmarks.

Reply:
That makes sense :)